src/elasticodbc/es_driver_connect.cpp

```diff
@@ -179,6 +179,10 @@ static SQLRETURN SetupConnString(const SQLCHAR *conn_str_in,
         return SQL_ERROR;
     }
 
+    //This will be used to restore the log output dir fetched from connection string
+    //Since getDSNinfo overrides all available connection attributes
+    std::string conn_log_dir(ci->drivers.output_dir);
```

Comment:
Can we name this to `conn_string_log_dir` or something similar to help highlight the point that this is from the connection string?

Reply:
renamed